src/native/eventpipe/ep-stream.c

```diff
@@ -696,6 +708,21 @@ ep_stream_writer_write (
 		bytes_written);
 }
 
+/*
+ * IpcContinuationStream.
+ */
+
+bool
+ep_ipc_continuation_stream_connection_closed (IpcContinuationStream *ipc_continuation_stream)
```

Comment:
```suggestion
ep_is_stream_connection_closed (IpcContinuationStream *ipc_continuation_stream)
```

Reply:
Since we can just invoke the `ep_ipc_stream_poll_vcall` directly on the session's continuation stream, just made this a static function in `ep-session.c` and renamed to `session_is_stream_connection_closed`.